ProjectShop/ProductApp/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from rest_framework.viewsets import ModelViewSet
+from rest_framework.response import Response
+from rest_framework.decorators import action
 
-# Create your views here.
+from ProductApp.models import Product, ProductCategory, ProductSubcategory, ProductMedia, Review, Tag
+from ProductApp.serializers import ProductSerializer, ProductCategorySerializer, ProductSubcategorySerializer
```

Comment:
Please change import from ProductApp.serializers

Reply:
imports changed, thanks for suggestion